packages/airnode-operation/src/config/evm-dev-config.json

```diff
@@ -17,7 +17,7 @@
             { "type": "bytes32", "name": "to", "value": "USD" },
             { "type": "bytes32", "name": "_type", "value": "int256" },
             { "type": "bytes32", "name": "_path", "value": "result" },
-            { "type": "bytes32", "name": "_times", "value": "100000" }
+            { "type": "bytes32", "name": "_times", "value": "1000000" }
```

Comment:
Not sure why but this value was not matching the config.json.example value https://github.com/api3dao/airnode/blob/5798e606b070d188344c70b5edb5ad68ad1e16d4/packages/airnode-node/config/config.json.example#L118